Breaking changes: Recommend checking error messages and documentation (#222)
bazelbuild/bazel-website: breaking-changes-guide.md

In the commit description, use [`RELNOTES:`](release-notes.html) followed by a short description, the
name of the flag, and a link to the GitHub issue.

The commit should also update the relevant documentation. As the documentation

> is versioned, it is recommended to update the documentation in the same commit

Reviewer: ```suggestion
is versioned, we recommend updating the documentation in the same commit
```

    name the of the flag.
  * Use `Fixes #xyz` in the description, so that the GitHub issue gets closed
    when the commit is merged.

>   * Review the documentation and update it if needed.

Reviewer: ```suggestion
  * Review and update documentation if needed.
```
